packages/pn-personagiuridica-webapp/src/components/IntegrazioneApi/VirtualKeyContextMenu.tsx

```diff
@@ -25,6 +25,7 @@ const VirtualKeyContextMenu: React.FC<Props> = ({ data, keys, handleModalClick }
   const currentUser = useAppSelector((state: RootState) => state.userState.user);
   const role = currentUser.organization?.roles ? currentUser.organization?.roles[0] : null;
   const isUserAdmin = useHasPermissions(role ? [role.role] : [], [PNRole.ADMIN]) && !currentUser.hasGroup;
+  const menuElement : Array<null | ReactNode> = [];
```

Comment:
Remove null and rename the constant 'menuElements' since it's a collaction of elements

Reply:
done